dpe/src/dpe_instance.rs

```diff
@@ -753,6 +810,51 @@ pub mod tests {
         assert_eq!(expected_index, idx);
     }
 
+    #[test]
+    fn test_add_tci_measurement() {
+        let mut dpe =
+            DpeInstance::<OpensslCrypto>::new_for_test(Support::default(), &TEST_LOCALITIES)
+                .unwrap();
+
+        // Verify bounds checking.
+        assert_eq!(
+            Err(DpeErrorCode::InternalError),
+            dpe.add_tci_measurement(MAX_HANDLES, &TciMeasurement::default())
+        );
+
+        let md = match DPE_PROFILE {
+            DpeProfile::P256Sha256 => openssl::hash::MessageDigest::sha256(),
+            DpeProfile::P384Sha384 => openssl::hash::MessageDigest::sha384(),
+        };
```

Comment:
nit: Could use OpensslCrypto hasher instead, just because it plays nicer with the DPE profile functions

Reply:
I originally thought to use plain OpenSSL to prove we were using the library correctly, but I think using `OpensslCrypto` is enough proof.

Done